lib/user.rb

```diff
@@ -0,0 +1,68 @@
+require 'pg'
+
+class User
+  attr_reader :username, :password, :email_address, :id
```

Comment:
Essure you restrict the access of the password as it's very visible in an attribute reader :)